examples/gallery/symbols/patterns.py

```diff
@@ -19,49 +19,47 @@
   ``uncertaintyfill``
 - :meth:`pygmt.Figure.wiggle`: Anomalies via ``fillpositive`` and ``fillnegative``
 
-GMT provides 90 predefined patterns that can be used in PyGMT. The patterns are numbered
-from 1 to 90, and can be colored and inverted. The resolution of the pattern
-can be changed, and the background and foreground colors can be set. For a complete list
-of available patterns and the full syntax to specify a pattern, refer to the
-:doc:`/techref/patterns`.
+GMT provides 90 predefined 1-bit patterns, which are numbered from 1 to 90. In addition,
+custom 1-, 8-, or 24-bit image raster files can also be used as patterns.
+
+Theses patterns can be specified via the :class:`pygmt.params.Pattern` class. The
+patterns can be customized with different resolution and different foreground and
+background colors. The foreground and background colors can also be inverted.
 """
 
 # %%
 import pygmt
+from pygmt.params import Pattern
 
 # A list of patterns that will be demonstrated.
-# To use a pattern as fill append "p" and the number of the desired pattern.
-# By default, the pattern is plotted in black and white with a resolution of 300 dpi.
+# By default, a pattern is plotted in black and white with a resolution of 1200 dpi.
```

Comment:
Did the default change from 300 dpi to 1200 dpi? Or was 300 dpi wrong / a typo?

Reply:
The upstream documentation says 1200, but the source code says 300. Actually it's a typo in the upstream documentation. See PR https://github.com/GenericMappingTools/gmt/pull/8789 for details.